src/app/models/test.type.ts

```diff
@@ -9,7 +14,7 @@ export interface TestType {
   numberOfSeatbeltsFitted: number;
   testTypeEndTimestamp: Date;
   reasonForAbandoning: string;
-  lastSeatbeltInstallationCheckDate: string;
+  lastSeatbeltInstallationCheckDate: Date;
```

Comment:
the API spec specifies this Date fields as string. Please lets obey that. Change where necessary to use string instead of the date type

Reply:
It is confusing because some of them have already **Date** as type. Look at **testTypeEndTimestamp** above. I agree that we should be consistent and I will change. Thanks for noticing